Tasks/VsTest/README.md

```diff
@@ -60,11 +66,12 @@ The feature is presently scoped to the following:
 
 #### Advanced Execution Options
 
-- **VSTest version:** Choose which version of Visual Studio (vstest.console.exe) to run tests with. 
+- **Batch tests:** Choose what type of batching to use to group tests. A batch is a group of tests. A batch of tests runs at a time and results are published for that batch. If the phase in which the task runs is set to use multiple agents, each agent picks up any available batches of tests to run in parallel.
```

Comment:
Start with 'A batch is..' Add 'Choose one of the below mentioned options for batching'.